National Parks/National Parks/Views/AuthError.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AuthError.swift
+//  National Parks
+//
+//  Created by Michael Koohang on 1/21/20.
+//  Copyright © 2020 Will Said. All rights reserved.
+//
+
+import SwiftUI
+
+struct AuthError: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .foregroundColor(Color.white)
+            .padding()
+            .background(Color.red)
+            .cornerRadius(10)
+            .padding(.leading, 20)
+            .padding(.trailing, 20)
+    }
+}
```

Comment:
this is the same view modifier as the existing AuthTextField view modifier, the only difference is the background color. So is this still needed?

Reply:
I wasn't able to change the background color after applying the modifier, so I just created independent view modifiers for each UI component.